crates/nargo_cli/tests/test_data_ssa_refactor/arithmetic_binary_operations/src/main.nr

```diff
@@ -1,12 +1,13 @@
+use dep::std;
 // Tests a very simple program.
 // 
 // The features being tested are:
 // Binary addition, multiplication, division
 // x = 3, y = 4, z = 5
 fn main(x : Field, y : Field, z : Field) -> pub Field {
-    let a = x + x; // 3 + 3 = 6
-    let b = a - y; // 6 - 4 = 2
-    let c = b * z; // 2 * 5 = 10
-    let d = c / a; // 10 / 6 (This uses field inversion, so we test it by multiplying by `a`)
-    d * a
+     let a = x + x; // 3 + 3 = 6
```

Comment:
The changes in this test can be reverted

Reply:
I don't see this one?

Reply:
Maybe you're hiding whitespace changes? There was an extra space added before each line in `main` and an extra `use dep::std` added on the first line.